packages/components/src/GuidedTour/GuidedTour.scss

```diff
@@ -30,9 +58,26 @@ $tc-guidedtour-close-button-size: $svg-sm-size !default;
 		.body {
 			margin: 0 0 $padding-larger 0;
 		}
+
+		.body {
```

Comment:
You can merge this one with the previous one.

Reply:
nop, be careful with collapsed diff :)

![image](https://user-images.githubusercontent.com/18534166/55316396-9554a900-546e-11e9-9a87-7946ff84e3aa.png)

Reply:
Oh, 👍